pkg/lib/form-layout.less

```diff
@@ -110,30 +111,38 @@
   }
 
   // Special considerations for widgets (and widget-like elements)
-  > input,
-  > textarea,
-  > select,
-  > .bootstrap-select,
-  > .dropdown,
-  > .combobox-container,
-  > fieldset,
-  > [role=group],
-  > .form-group,
-  > .btn-group,
-  > .checkbox-inline,
-  > .radio-inline {
-    // Vertically align elements with text
-    margin-top: -(@padding-small-vertical - @padding-base-vertical);
-    // Ensure widgets are a at least a minimum widget size
-    min-height: @widget-height;
-
-    // Some elements need special width considerations
-    // as PatternFly normally fixes the width
-    &:not(.ct-form-layout-relax) {
-      width: auto !important;
+  // This is a LESS mixin that will not be in the compiled CSS.
+  .widget-rules() {
+    > input,
+    > textarea,
+    > select,
+    > .bootstrap-select,
+    > .dropdown,
+    > .combobox-container,
+    > fieldset,
+    > [role=group],
+    > [data-field],
+    > .form-group,
+    > .btn-group,
+    > .checkbox-inline,
+    > .radio-inline {
+      // Vertically align elements with text
+      margin-top: -(@padding-small-vertical - @padding-base-vertical);
+      // Ensure widgets are a at least a minimum widget size
+      min-height: @widget-height;
     }
   }
 
+  &, > .ct-validation-wrapper {
```

Comment:
This exports the less mixin above to insert `> .ct-validation-wrapper` in between the top-level grid and the elements.

Reply:
I was playing with all sorts of LESS code. I could do this in SASS already, but haven't done this more complex LESS trickery.

I don't want to duplicate that selector list anywhere else, as it would easily get out of sync and such.